frontend/packages/knative-plugin/integration-tests/features/kafka-event-source/eventing-kafka-event-source.feature

```diff
@@ -4,8 +4,9 @@ Feature: Kafka Event Source
 
 
         Background:
-            Given Kafka Operator installed
-              And BootStrapServers and Topics created
+            Given user has installed Serverless Operator
+              And user has created kafka instance in knative-eventing namespace
+              And user has created BootStrapServers and Topics
```

Comment:
Not sure what `created BootStrapServers` mean

Reply:
should this be `User has created Kafka instance and Topics`?

Reply:
Updated to `User has created Kafka instance and Topics`